### DISCUSSION TITLE
API - fetching all Merge Requests that were merged between 2 dates

### DISCUSSION THREAD

**Discussion by rocodcli** [GitbookIO/community#402] [0. General bugs and questions]
Hello,

I would like to keep track of the new content in out gitbook on a regular basis, I would like to use the API to fetch all the merge requests that were merged between 2 dates, is it possible to do so today?

**Comment by rocodcli** [comment]
Any help with that?

**Reply by aikoii** [reply]
👋 @rocodcli! Currently, our API doesn't support filtering merge requests by date. However, you can use the `updatedAt` field to track merged change requests. This field can act as `mergedAt` if there haven't been further updates post-merge. Remember, you'll need to filter the API calls with `status=merged` and manually 🙈 sort through the full list of CRs, as date-specific filtering isn't available yet.

For more details, check our [GitBook API documentation](https://developer.gitbook.com/gitbook-api/reference/spaces/change-requests).

We're definitely looking into adding date-range filtering! But I don't have a timeline on this at the moment.
